src/controllers/profile.js

```diff
@@ -0,0 +1,25 @@
+import responseUtil from '../utils/responseUtil';
+import responseError from '../utils/responseError';
+import string from '../utils/stringsUtil';
+import models from '../database/models';
+
+const { users } = models;
+
+export default class profile {
+  static async updateProfile(req, res) {
+    try {
+      const newUser = await users.update(
+        req.body,
+        {
+          where: { email: req.params.email }, returning: true, plain: true
+        }
+      );
+      const {
+        password, createdAt, updatedAt, isVerified, ...data
+      } = newUser[1].dataValues;
+      return responseUtil(res, 200, string.users.success.SUCCESS_UPDATE, data);
+    } catch (error) {
+      return responseError(res, 500, string.users.error.SOMETHING_WRONG);
+    }
+  }
+}
```

Comment:
Please respect naming conventions when naming files. I think this file should be profileController.js